lib/gateways/fixed_data_gateway.rb

```diff
@@ -0,0 +1,9 @@
+class FixedDataGateway
+  def find_by_year(year)
+    FixedDatum.find_by(year: year)
```

Comment:
Are you planning to use the Domain::FixedData object here? and in `rc_uplift`? That way the gateway will return a standard object, rather than a Rails specific object.